src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        String color = supplier.getRandomColor();
+        int num = new Random().nextInt(100);
+
+        return new Ball(color, num);
```

Comment:
You can directly return the new Ball instance without creating temporary variables for color and num.